docs/content/server/new_blocks.md

```diff
@@ -21,10 +23,11 @@ To create a new block, follow these steps:
    Example for the Wikipedia summary block:
 
    ```python
-   import requests
    from autogpt_server.data.block import Block, BlockSchema, BlockOutput
+   from autogpt_server.utils.get_request import GetRequest
+   import requests
 
-   class GetWikipediaSummary(Block):
+   class WikipediaSummaryBlock(Block, GetRequest):

```

Comment:
Maybe we need simpler example where we don't extend `GetRequest` this is just my way to cleanup repetitive code and easier mocking. Should we do HttpRequestBlock ?

Reply:
Yeah i think that might be a better idea, ill get this changed out to use the HttpRequestBlock

Reply:
It may actually be useful to keep this as is. IMO this is a really valid example of what to do. Maybe as an advanced page on "extending an existing block"